src/LogRequest.php

```diff
@@ -24,21 +24,29 @@ public function handle($request, Closure $next)
     {
 
         if (! $this->isExceptRequest($request)) {
-            (new RequestLogEntry)->forceFill([
-                'ip' => $request->getClientIp(),
-                'path' => urldecode($request->path()),
-                'method' => $request->getMethod(),
-                'agent' => $request->server('HTTP_USER_AGENT'),
-                'get' => $this->get($request),
-                'post' => $this->post($request),
-                'cookies' => $this->cookies($request),
-                'session' => $this->session($request)
-            ])->save();
+            RequestLogEntry::create($this->getData());
         }
 
         return $next($request);
     }
 
+    /**
+     * @return array
+     */
+    protected function getData() : array
```

Comment:
It seems the composer.json is missing some important information. Due to the return types the package will only work with php < 7. Can you please add the php requirements to composer.json?

Reply:
https://github.com/doctrine/dbal/blob/v2.9.0/composer.json#L24-L27

Reply:
Fair enough didn't have that in mind.